src/desktop/apps/rss/templates/news.jade

```diff
@@ -13,6 +13,12 @@ rss(version='2.0', xmlns:atom='http://www.w3.org/2005/Atom' xmlns:content='http:
         title= article.get('thumbnail_title')
         link #{sd.APP_URL}/article/#{article.get('slug')}
         description= article.get('description')
+        author=article.byline() || "Artsy Editors"
+        if article.get('vertical')
+          category=article.get('vertical').name
+        if article.get('layout') === 'video'
+          - media = article.get('media')
+          enclosure(url=media.url type="video/mp4")
         content:encoded
```

Comment:
The video is also rendered in the `articles.jade` file, or is this some other kind of meta?

Reply:
Yep, this is a field that exists outside of the article content! But this reminds me that we should use thumbnail images for regular articles as the enclosure.